### DISCUSSION TITLE
Welcome to the Discussion!

### DISCUSSION THREAD

**Discussion by stevenilsen123** [stevenilsen123/mac-keyboard-behavior-in-windows#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by patrickhan93** [comment]
Hi Steve, thanks so much for building this, it's super helpful! I was wondering, is there an easy way to undo the script to go back to regular PC keyboard besides rebooting the computer? Thanks!

**Reply by stevenilsen123** [reply]
Hi @patrickhan93, sorry for the delayed response. Yes! You can pause the script at any time by right-clicking the Autohotkey icon in the system tray (bottom-right).
